packages/client/ui/react-ui/src/components/auth/AuthForm.tsx

```diff
@@ -9,6 +9,7 @@ import { PoweredByCrossmint } from "../common/PoweredByCrossmint";
 import { FarcasterProvider } from "../../providers/auth/FarcasterProvider";
 import { classNames } from "@/utils/classNames";
 import { AlertIcon } from "@/icons/alert";
+import { XSignIn } from "./methods/x/XSignIn";
```

Comment:
if we're using the name twitter everywhere, I would stick to that, or rename to X everywhere

Reply:
nice catch, i renamed everything to twitter except for the button itself

>Sign in with X